packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -218,45 +155,12 @@ export interface AddRoutesOptions extends BatchHttpRouteOptions {
   readonly authorizationScopes?: string[];
 }
 
-abstract class HttpApiBase extends Resource implements IHttpApi { // note that this is not exported
+abstract class HttpApiBase extends ApiBase implements IHttpApi { // note that this is not exported
```

Comment:
Do we need `HttpApiBase` anymore? Can we just use `ApiBase`?

Reply:
We still need it because it has public method `addVpcLink` that can be potentially used for imported APIs as well. To remove it, we will have to duplicate it in the `from*` helpers.

Reply:
Ok.